corticalmapping.py

```diff
@@ -18,7 +18,7 @@
 import numpy as np
 import openmeeg as om
 from om_basics  import om_load_headmodel, om_forward_problem # openmeeg basics
-from om_display import om_display_vtp # visualiation with VTK
+from openmeeg_viz.om_display import om_display_vtp # visualiation with VTK
```

Comment:
you could add the om_display_vtp to the `openmeeg_viz/__init__.py` to be able to do:

from openmeeg_viz import om_display_vtp

also better would be rename om_display_vtp to display_vtp as openmeeg is already in the import name.